lxd/openfga/model.go

```diff
@@ -0,0 +1,5 @@
+package openfga
+
+// Code generated by Makefile; DO NOT EDIT.
```

Comment:
Could we change this to "Code generated by make update-openfga` so its clear what needs to be run to update it?